ui/app/hooks/useSwapsEthToken.js

```diff
@@ -0,0 +1,42 @@
+import { useSelector } from 'react-redux'
+import { getSelectedAccount } from '../selectors'
+import { ETH_SWAPS_TOKEN_OBJECT } from '../helpers/constants/swaps'
+import { getValueFromWeiHex, hexToDecimal } from '../helpers/utils/conversions.util'
+
+/**
+ * @typedef {Object} SwapsEthToken
+ * @property {string} decimals - a secondary currency string formatted for display
+ * @property {string} symbol - the symbol for ETH, namely "ETH"
+ * @property {string} name - the name of the eth current, "Ether"
+ * @property {string} address - a substitute address for the metaswap-api to recognize the eth token
+ * @property {string} decimals - the number of decimal places of divisibility of eth, 1 ETH is represented
+ *  by 10^18 of its natural unit ( 1 Ether = 1,000,000,000,000,000,000 wei ). Hence, this will be 18.
+ * @property {string} balance - the users ETH balance in decimal WEI
+ * @property {string} string - the users ETH balance in decimal ETH
```

Comment:
Might be worth specifying the rounding too:
```suggestion
 * @property {string} string - the users ETH balance in decimal ETH, rounded to 4 decimals
```